queries/scala/highlights.scm

```diff
@@ -110,20 +122,27 @@
 (floating_point_literal) @float
 
 [
-(symbol_literal)
-(string)
-(character_literal)
-(interpolated_string_expression)
+  (symbol_literal)
+  (string)
+  (character_literal)
+  (interpolated_string_expression)
 ] @string
 
 (interpolation "$" @punctuation.special)
 
 ;; keywords
 
+(opaque_modifier) @type.qualifier
+(infix_modifier) @keyword
+(transparent_modifier) @type.qualifier
+(open_modifier) @type.qualifier
```

Comment:
These could all be `@type.qualifier` despite `infix` not necessarily applying to types.
(The plan is to rename it to `@modifier` or something eventually so it makes more sense.)

Reply:
Just to clarify, so these _are_ ok right now? Or you want `infix` changed? I sort of like infix being a keyword here at the moment, since like you mentioned, it's not really applying to `type` at all.

Reply:
Yes, they're fine.